build-system/compile/bundles.config.js

```diff
@@ -529,6 +529,12 @@ exports.extensionBundles = [
     options: {hasCss: true},
     type: TYPES.MISC,
   },
+  {
+    name: 'amp-fit-text',
```

Comment:
I think this is supposed to be joined with the other `amp-fit-text`, just add a `version: ['0.1', '0.2']` to that one. Right?

Reply:
We need a separate config entry for this because `options.hasCss` is different. :/